invenio_app_ils/circulation/serializers/utils.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Loan custom serializer functions."""
+
+from flask import current_app
+from invenio_pidstore.errors import PIDDeletedError, PIDDoesNotExistError
+
+from invenio_app_ils.circulation.utils import circulation_overdue_loan_days
+from invenio_app_ils.jsonresolver.api import pick
+from invenio_app_ils.proxies import current_app_ils
+
+
+def add_is_overdue(metadata):
```

Comment:
Suggestions for the naming of these methods? And for the name of the file instead of `utils`?

Reply:
`field_is_overdue` for the methods name?
and then `custom_fields.py` or similar for the file name